### PR TITLE
cd: add initial Dockerfile

### PR DESCRIPTION
**Pull request by LenaIkra**
I've added initial Dockerfile. Please, review if it works OK and then we can merge it.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,28 @@
+FROM ubuntu:latest
+
+LABEL org.opencontainers.image.source https://github.com/mlteamurfu2325/practicum-s1
```
Comment (khit-mle) [inline Dockerfile:3]: Apart from adding `LABEL` to link our image to repo, I think we should also add this from GHCR.io's docs:
```
LABEL org.opencontainers.image.description DESCRIPTION
```

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+ENV DIR_NAME=practicum
+ENV GIT_BRANCH=main
+ENV LLM_SUMMARIZER=n
+ENV RUN_APP=n
+
+# Run the installation script with the provided arguments
+RUN ./install.sh --dir_name=$DIR_NAME --git_branch=$GIT_BRANCH --llm_summarizer=$LLM_SUMMARIZER --run_app=$RUN_APP
+
+# Set the default command to run when the container starts
+CMD ["/bin/bash", "-c", "source /root/practicum/.venv-practicum/bin/activate && cd /root/practicum/src && streamlit run run_app.py"]
```
Comment (khit-mle) [inline Dockerfile:28]: A bit hack-y, but I think it's OK for MVP ✅

**Review by khit-mle — CHANGES_REQUESTED** [review]
Let's also try to implement a GitHub workflow that would build a new image upon a new PR merged into our `main`.

**Review by ANGorbachev — CHANGES_REQUESTED** [review]
I agree with @khit-mle. We need to write a GitHub workflow to automate building a new Docker image for every PR merged into `main`.

**Pull request merged**